ledger/src/proofs/caching.rs

```diff
@@ -98,12 +107,12 @@ pub struct GroupAffineCached {
     infinity: bool,
 }
 
-impl<'a, T> From<&'a GroupAffine<T>> for GroupAffineCached
+impl<'a, T> From<&'a ark_ec::models::short_weierstrass::Affine<T>> for GroupAffineCached
```

Comment:
Nit: `Affine` could be used directly, to avoid full path. We should try to enforce one code convention in the entire codebase.